### PR TITLE
Inline js runtime bundle

### PR DESCRIPTION
**Pull request by yishengjiang99**
screenshot-pr-1661

### CODE REVIEW

File: packages/subapp-web/lib/init.js [hunk 1]
```diff
@@ -34,10 +30,18 @@ module.exports = function setup(setupContext) {
     basePath: ""
   };
 
+  const approot = process.env.APP_ROOT_DIR || Path.resolve() || process.cwd();
```
Comment (jchip) [inline packages/subapp-web/lib/init.js:33]: - don't use env
- `Path.resolve("dist")` should do

File: packages/subapp-web/lib/init.js [hunk 2]
```diff
@@ -34,10 +30,18 @@ module.exports = function setup(setupContext) {
...
+  let inlineRuntimeJS = "";
+  const runtimeFSPath = cdnJsBundles.runtime && Path.join(approot, "dist", cdnJsBundles.runtime);
+  if (runtimeFSPath && Fs.existsSync(runtimeFSPath)) {
```
Comment (jchip) [inline packages/subapp-web/lib/init.js:36]: only do this for `NODE_ENV` === `production`

File: packages/subapp-web/test/data/mock-app/dist/stats-with-runtime.json [hunk 3]
```diff
@@ -0,0 +1,107 @@
+{
```
Comment (jchip) [inline packages/subapp-web/test/data/mock-app/dist/stats-with-runtime.json:1]: you need to use stats from production build.  get it from `dist/server` by running `clap build`

**Review by jchip — CHANGES_REQUESTED** [review]

**Comment by yishengjiang99** [comment]
screenshot-pr-1661

**Pull request closed without merging**